source/mesh_deformation/interface.cc

```diff
@@ -665,7 +665,7 @@ namespace aspect
       fs_mesh_velocity.reinit(mesh_locally_owned, mesh_locally_relevant, sim.mpi_communicator);
```

Comment:
Specifically, in line 664 (two lines up, but I can't annotate that line :-) ) `mesh_displacements` is set to zero. In other words, the call you are modifying should not actually have any discernible consequence. If you're wondering, you could just put
```
  Assert (mesh_displacement.l2_norm() == 0, ExcInternalError());`
```
here and verify that that is always satisfied. I believe that you can completely remove the two lines below.